## Single Cell
<table>
    <tr>
        <td colspan="6" align="center"><strong>Hi-C</strong></td>
    </tr>
    <tr>
        <td align="center">sciHi-C</td> 
        <td align="center"><a href="https://www.nature.com/articles/nmeth.4155">Massively multiplex single-cell Hi-C</a></td> 
        <td align="center">Nature Method</td> 
        <td align="center">2017-01-30</td> 
        <td align="center"><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84920">Data_part1</a> <a href="https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000640.v4.p1">Data_part2</a></td>
        <td align="center"><a href="https://github.com/VRam142/combinatorialHiC">Code</a></td>
    </tr>
</table>


## Single Cell Multi Omics Technologies

<table>
    <tr>
        <td colspan="6" align="center"><strong>Transcriptome & chromotin accessbility</strong></td>
    </tr>
    <tr>
        <td align="center" width="110">scCAT-seq</td> 
        <td align="center"><a href="https://www.nature.com/articles/s41467-018-08205-7">Deconvolution of single-cell multi-omics layers reveals regulatory heterogeneity</a></td> 
        <td align="center" width="60">Nature Communications</td> 
        <td align="center" width="120">2019-01-28</td> 
        <td align="center"><a href="https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?study=SRP136421">Data_NCBI_SRA</a> <a href="https://db.cngb.org/cnsa/project/CNP0000213/public/">Data_CNGB</a></td>
        <td align="center"></td>
    </tr>
    <tr>
        <td align="center">sci‐CAR</td> 
        <td align="center"><a href="https://science.sciencemag.org/content/361/6409/1380.full">Joint profiling of chromatin accessibility and gene expression in thousands of single cells</a></td> 
        <td align="center">Science</td> 
        <td align="center">2019-09-28</td> 
        <td align="center"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE117089">Data_NCBI_GEO</a></td>
        <td align="center"><a href="https://github.com/JunyueC/sci-CAR_analysis">Code</a></td>
    </tr>
    <tr>
        <td align="center">SNARE‐seq</td> 
        <td align="center"><a href="https://www.nature.com/articles/s41587-019-0290-0">High-throughput sequencing of the transcriptome and chromatin accessibility in the same cell</a></td> 
        <td align="center">Nature Biotechnology</td> 
        <td align="center">2019-10-14</td> 
        <td align="center"><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE126074">Data_NCBI_GEO</a></td>
        <td align="center"><a href="https://github.com/chensong611/SNARE_prep">Code</a></td>
    </tr>
    <tr>
        <td align="center">ASTAR-Seq</td> 
        <td align="center"><a href="https://genome.cshlp.org/content/30/7/1027.full">Parallel Bimodal Single-cell Sequencing of Transcriptome and Chromatin Accessibility</a></td> 
        <td align="center">Genome Research</td> 
        <td align="center">2020-07-22</td> 
        <td align="center"><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE113418">Data_NCBI_GEO</a></td>
        <td align="center"><a href="https://genome.cshlp.org/content/suppl/2020/07/22/gr.257840.119.DC1/Supplemental_Codes.zip">Code</a></td>
    </tr>
    <tr>
        <td align="center">Paired-seq</td> 
        <td align="center"><a href="https://www.nature.com/articles/s41594-019-0323-x">An ultra high-throughput method for single-cell joint analysis of open chromatin and transcriptome</a></td> 
        <td align="center">Nature Structural & Molecular Biology</td> 
        <td align="center">2019-11-06</td> 
        <td align="center"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130399">Data_NCBI_GEO</a></td>
        <td align="center"><a href="https://github.com/cxzhu/paired-seq">Code</a></td>
    </tr>
    <tr>
        <td colspan="6" align="center"><strong>Transcriptome & Proteome</strong></td>
    </tr>
    <tr>
        <td align="center">PLAYR</td> 
        <td align="center"><a href="https://www.nature.com/articles/nmeth.3742">Highly multiplexed simultaneous detection of RNAs and proteins in single cells</a></td> 
        <td align="center">Nature Methods</td> 
        <td align="center">2016-01-25</td> 
        <td align="center"></td>
        <td align="center"></td>
    </tr>
    <tr>
        <td align="center">Abseq</td> 
        <td align="center"><a href="https://www.nature.com/articles/srep44447">Abseq: Ultrahigh-throughput single cell protein profiling with droplet microfluidic barcoding</a></td> 
        <td align="center">Scientific Reports</td> 
        <td align="center">2017-03-14</td> 
        <td align="center"></td>
        <td align="center"></td>
    </tr>
    <tr>
        <td align="center">REAP-seq</td> 
        <td align="center"><a href="https://www.nature.com/articles/nbt.3973">Multiplexed quantification of proteins and transcripts in single cells</a></td> 
        <td align="center">Nature Biotechnology</td> 
        <td align="center">2017-08-30</td> 
        <td align="center"><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE100501">Data_NCBI_GEO</td>
        <td align="center"><a href="https://github.com/kelvinxz/RS_aligner">Code</td>
    </tr>
    <tr>
        <td align="center">CITE-seq</td> 
        <td align="center"><a href="https://www.nature.com/articles/s41592-019-0392-0">Multiplexed detection of proteins, transcriptomes, clonotypes and CRISPR perturbations in single cells</td> 
        <td align="center">Nature Methods</td> 
        <td align="center">2019-04-22</td> 
        <td align="center"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE126310">Data_NCBI_GEO</td>
        <td align="center"><a href="https://github.com/Hoohm/CITE-seq-Count">Code</td>
    </tr>
    <tr>
        <td colspan="6" align="center"><strong>Transcriptome & Methylome</strong></td>
    </tr>
    <tr>
        <td align="center">scM&T-seq</td> 
        <td align="center"> <a href="https://www.nature.com/articles/nmeth.3728">Parallel single-cell sequencing links transcriptional and epigenetic heterogeneity</a></td> 
        <td align="center">Nature Methods</td> 
        <td align="center">2016-01-11</td> 
        <td align="center"></td>
        <td align="center"><a href="https://github.com/PMBio/scMT-seq.git">Code</a></td>
    </tr>
    <tr>
        <td colspan="6" align="center"><strong>Methylation & Chromatin accessibility & CNV</strong></td>
    </tr>
    <tr>
        <td align="center">scCOOL-seq</td> 
        <td align="center"> <a href="https://www.nature.com/articles/s41556-018-0123-2">Single-cell multi-omics sequencing of human early embryos</a></td> 
        <td align="center">Nature Cell Biology</td> 
        <td align="center">2018-06-18</td> 
        <td align="center"><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE100272">Data_part1</a> <a href="https://www.ebi.ac.uk/ega/studies/EGAS00001002987">Data_part2</a></td>
        <td align="center"></td>
    </tr>
    <tr>
        <td colspan="6" align="center"><strong>Transcriptome & Methylome & CNV</strong></td>
    </tr>
    <tr>
        <td align="center">scTrio‐seq</td> 
        <td align="center"> <a href="https://science.sciencemag.org/content/362/6418/1060.full">Single-cell multiomics sequencing and analyses of human colorectal cancer</a></td> 
        <td align="center">Science</td> 
        <td align="center">2018-11-30</td> 
        <td align="center"><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE97693">NCBI_GEO</a></td>
        <td align="center"><a href="https://github.com/bianshuhui/CRC_code">Code</a></td>
    </tr>
    <tr>
        <td colspan="6" align="center"><strong>Transcriptome & Methylation & Chromatin accessibility</strong></td>
    </tr>
    <tr>
        <td align="center">scNMT‐seq</td> 
        <td align="center"><a href="https://www.nature.com/articles/s41467-018-03149-4">scNMT-seq enables joint profiling of chromatin accessibility DNA methylation and transcription in single cells</a></td> 
        <td align="center">Nature Communications</td> 
        <td align="center">2018-02-22</td> 
        <td align="center"><a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=">Data_NCBI_GEO</a></td>
        <td align="center"><a href="https://github.com/PMBio/scNMT-seq/">Code</a></td>
    </tr>
    <tr>
        <td align="center">scChaRM-seq</td> 
        <td align="center"><a href="https://www.sciencedirect.com/science/article/pii/S1934590921001697">Decoding dynamic epigenetic landscapes in human oocytes using single-cell multi-omics sequencing</a></td> 
        <td align="center">Cell Stem Cell</td> 
        <td align="center">2021-05-05</td> 
        <td align="center"><a href="https://bigd.big.ac.cn/search/?dbId=&q=HRA000787">Data_GSA_NGDC</a> <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE154762">Data_NCBI_GEO</a></td>
        <td align="center">No Code</td>
    </tr>
</table>


